apps/editing-toolkit/package.json

```diff
@@ -40,10 +40,8 @@
 		"lint:php": "../../vendor/bin/phpcs --standard=../phpcs.xml ./ ",
 		"lint:php:fix": "../../vendor/bin/phpcbf --standard=../phpcs.xml ./ ",
 		"sync:newspack-blocks": "./bin/sync-newspack-blocks.sh",
-		"test:js": "npx wp-scripts test-unit-js --config='jest.config.js' --colors",
-		"test:js:help": "npx wp-scripts test-unit-js --config='jest.config.js' --help --colors",
-		"test:js:update-snapshots": "npx wp-scripts test-unit-js -u --config='jest.config.js' --colors",
```

Comment:
I don't think these are useful -- `yarn test:js -u`, or `yarn test:js --help` will access these options. --watch would also work, but it's more commonly used.